From a19e829e674a52dc9cad535be9dd56e651527e56 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <bonzini@gnu.org>
Date: Wed, 29 Jul 2009 16:28:56 +0200
Subject: [PATCH] Add back AH_CHECK_HEADERS.

* lib/autoconf/general.m4 (AH_CHECK_HEADERS): New.
* NEWS: Add new section.
---
 ChangeLog               |    6 ++++++
 NEWS                    |    9 +++++++++
 lib/autoconf/headers.m4 |    6 ++++++
 3 files changed, 21 insertions(+), 0 deletions(-)

diff --git a/lib/autoconf/headers.m4 b/lib/autoconf/headers.m4
index 21b512c..36c1899 100644
--- a/lib/autoconf/headers.m4
+++ b/lib/autoconf/headers.m4
@@ -253,6 +253,12 @@ m4_define([_AH_CHECK_HEADER],
   [Define to 1 if you have the <$1> header file.])])
 
 
+# AH_CHECK_HEADERS(HEADER-FILE...)
+# --------------------------------
+m4_define([AH_CHECK_HEADERS],
+[m4_foreach_w([AC_Header], [$1], [_AH_CHECK_HEADER(m4_defn([AC_Header]))])])
+
+
 # AC_CHECK_HEADERS(HEADER-FILE...,
 #		   [ACTION-IF-FOUND], [ACTION-IF-NOT-FOUND],
 #		   [INCLUDES])
-- 
1.6.3.3

